charts/internal/seed-controlplane/charts/cloud-provider-equinix-metal/values.yaml

```diff
@@ -11,4 +11,4 @@ resources:
   limits:
     cpu: 500m
     memory: 512Mi
-metro: ny
+metro: ewr1
```

Comment:
This is just an example value and changing it here will not have any effect. The `controlplane` controller does use `.spec.region` of the `Shoot` to fill this field, see https://github.com/gardener/gardener-extension-provider-equinix-metal/blob/53d9c1d0805d82ae46cf7a4221fb427389946604/pkg/controller/controlplane/valuesprovider.go#L180

Reply:
got it.
But then the `cluster.Shoot.Spec.Region` ist not the value that is expected, but one of the zones used by a worker.

Reply:
According to @deitch, the CCM can doesn't care much about the value in this field field as it's just passed on, so that why I implemented it this way:

```
rfranzke  1 month ago
Today, we are configuring CCM with .spec.region as METAL_FACILITY. However, according to my understanding, you want me to change this in the future such that
.spec.region is no longer a facility but a metro (always!)
.spec.provider.workers[].zones[] is a list of one or more facilities (at least one).
Under this assumption, which value should be used for METAL_FACILITY?

deitcher  1 month ago
Right. So given the following two things:
we insist that all zones must be in the region
when creating an EIP, it doesn’t matter if you include a metro or a facility, it always will give one valid for any facility in that metro
then we can give either. The smarter one to give is the metro. So we should continue to pass .spec.region , even (or especially) as it is now metro, and it should work correctly.

deitcher  1 month ago
As the CCM evolves, we may need to make minor changes here, but that is the thrust that we should follow

deitcher  1 month ago
Does that make sense?

rfranzke  1 month ago
Sure, sounds great, if today’s CCM can already understand metro values for METAL_FACILITY then it’s all good

deitcher  1 month ago
It doesn’t, but it is neutral about it, just passes it on. The EQXM API can handle the strings, so all good
```

Reply:
I agree. This should be the metro/region, not the facility/zone. 

As for the CCM using metadata, you are 100% correct. It can be overridden, but we plan on getting rid of dependence on metadata entirely

Reply:
I received a `ny 422` error in the logs with `ny`, which disappeared and worked when using `ewr1`. 
I am not able to reproduce it. But I am also not getting a working setup with `ny` ... nor with `ewr1` working again.

will close this PR/issue